pysmartthings/device.py

```diff
@@ -185,6 +201,13 @@ def is_on(self, attribute: str) -> bool:
             return bool(self._attributes[attribute].value)
         return self._attributes[attribute].value == ATTRIBUTE_ON_VALUES[attribute]
 
+    @staticmethod
```

Comment:
There's no need for this to be a static method given it doesn't reference anything within the class.  Let's make this a module level function.